RELEASE/scripts/autoscend/quests/level_any.ash

```diff
@@ -891,3 +891,93 @@ boolean LX_ForceNC()
+	{
+		auto_log_info("Going to Middle Chamber");
+		if(get_property("auto_priorLocation").to_location() != $location[The Middle Chamber])
+		{
+			set_property("auto_skipStage4", true);  //don't set skipStage2 because rat king
+		}
+		return autoAdv($location[The Middle Chamber]); //Tomb ratchets
+	}
+	if((internalQuestStatus("questL09Topping") >= 2 && internalQuestStatus("questL09Topping") <= 3) && get_property("twinPeakProgress").to_int() < 15 && zone_isAvailable($location[Twin Peak]))
+	{
+		auto_log_info("Going to Twin Peak");
+		if(get_property("auto_priorLocation").to_location() != $location[Twin Peak])
+		{
+			set_property("auto_skipStage2", true);
+			set_property("auto_skipStage4", true);
+		}
+		return autoAdv($location[Twin Peak]); //Hedge trimmers
+	}
+	if (internalQuestStatus("questL11Ron") > 1 && internalQuestStatus("questL11Ron") < 5 && zone_isAvailable($location[The Red Zeppelin]))
+	{
+		auto_log_info("Going to the Red Zeppelin");
+		if(get_property("auto_priorLocation").to_location() != $location[The Red Zeppelin])
+		{
+			set_property("auto_skipStage4", true); //don't set skipStage2 because glark cables
+		}
+		return autoAdv($location[The Red Zeppelin]); //Glark cables
+	}
+	if(canExtingo = false && (get_property("hiddenBowlingAlleyProgress").to_int() + item_amount($item[Bowling Ball])) < 6 && zone_isAvailable($location[The Hidden Bowling Alley]))
+	{
+		auto_log_info("Going to the Hidden Bowling Alley");
+		if(get_property("auto_priorLocation").to_location() != $location[The Hidden Bowling Alley])
+		{
+			set_property("auto_skipStage2", true);
+			set_property("auto_skipStage4", true);
+		}
+		return autoAdv($location[The Hidden Bowling Alley]); //Bowling balls
+	}
+	if(internalQuestStatus("questL04Bat") <= 1 && zone_isAvailable($location[The Batrat and Ratbat Burrow]))
+	{
+		auto_log_info("Going to the Batrat and Ratbat Burrow");
+		if(get_property("auto_priorLocation").to_location() != $location[The Batrat and Ratbat Burrow])
+		{
+			set_property("auto_skipStage2", true);
+			set_property("auto_skipStage4", true);
+		}
+		return autoAdv($location[The Batrat and Ratbat Burrow]); //Sonar-in-a-Biscuit
+	}
+	if(internalQuestStatus("questL08Trapper") == 1 && zone_isAvailable($location[The Goatlet]))
+	{
+		auto_log_info("Going to the Goatlet");
+		if(get_property("auto_priorLocation").to_location() != $location[The Goatlet])
+		{
+			set_property("auto_skipStage2", true);
+			set_property("auto_skipStage4", true);
+		}
+		return autoAdv($location[The Goatlet]); //Goat cheese
+	}
+	if(item_amount($item[Stone Wool]) == 0 && have_effect($effect[Stone-Faced]) == 0 && canSummonMonster($monster[Baa\'baa\'bu\'ran]) && internalQuestStatus("questL11Worship") < 3 && my_level() >= 11)
+	{
+		auto_log_info("Summoning Baa baa buran");
+		return summonMonster($monster[Baa\'baa\'bu\'ran]); //Stone wool
+	}
+	return false;
+}
```

Comment:
This file does not end in a new line.
files need to end in new line to prevent potential problems when they are imported